docs/tutorials/sso.md

```diff
@@ -51,22 +51,26 @@ If your LDAP does not provide anonymous authentication access, then you need to
 
 ## Platform setting
 
-Users who login in through SSO will be automatically bound to the users who have the same email in the local database, otherwise a new user will be created.
+Users who log in through SSO will be automatically bound to the users who have the same email in the local database, otherwise, a new user will be created.
```

Comment:
```suggestion
Users who logged in through SSO will be automatically bound to the users who have the same email in the local database, otherwise, a new user will be created.
```